src/TasksModule/Components/TasksList/TasksList.tsx

```diff
@@ -1,7 +1,148 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import { baseUrl } from "../../../Constants/Components/Urls.js";
+import axios from "axios";
+import { toast } from "react-toastify";
+import { useNavigate } from "react-router-dom";
+import NoData from "../../../noData/noData.js";
 
 export default function TasksList() {
+  const [taskList, setTaskList] = useState([]);
+  const [titleSearch, setTitleSearch] = useState("");
+  const [statusSearch, setStatusSearch] = useState("");
+  const [pagesArray, setPagesArray] = useState([]);
+
+  const token = localStorage.getItem("userToken");
+  const getTitleValue = (input: any) => {
```

Comment:
The name of this function is a little bit confusing, as it is responsible for updating the title that we use for searching and getting the tasks list as well.

I suggest changing it to something such as `searchByTitle`.
I also suggest taking the title as a parament not the complete input and then accessing it by calling `input.target.value`.

`searchByTitle(title:string)`

and when calling the function in the input => `searchByTitle(e.target.value)`